docs/source/guides/logging.rst

```diff
@@ -1,7 +1,154 @@
-Request Logging
-===============
+Logging
+=======
 
-This part of the BentoML documentation is a work in progress. If you have any questions
-related to this, please join
-`the BentoML Slack community <https://join.slack.com/t/bentoml/shared_invite/enQtNjcyMTY3MjE4NTgzLTU3ZDc1MWM5MzQxMWQxMzJiNTc1MTJmMzYzMTYwMjQ0OGEwNDFmZDkzYWQxNzgxYWNhNjAxZjk4MzI4OGY1Yjg>`_
-and ask in the bentoml-users channel.
\ No newline at end of file
+BentoML uses standard `Python logging <https://docs.python.org/3/howto/logging.html>`_ libraries and provides basic 
+logging customization through `logging` section in `bentoml.cfg` under the BentoML home directory. Refer to the 
+:ref:`configuration guide <configuration-page>` on how override configuration properties. See 
+`default_bentoml.cfg <https://github.com/bentoml/BentoML/blob/master/bentoml/configuration/default_bentoml.cfg>`_
+for a list of override-able properties.
+
+For advanced logging customization, user can provide full logging configurations in `logging.yml`, placed under 
+the BentoML home directory. For example, `logging.yml` configuration file can be injected into the Docker container 
+through the following command.
+
+.. code-block:: shell
+
+    $ docker run -v /local/path/to/logging.yml:{BENTOML_HOME}/logging.yml
+
+Please see below configuration examples of different logging scenarios in YAML format.
+
+.. code-block:: yaml
+    :caption: Enable INFO+ console logging but only WARN+ file logging.
+
+    version: 1
+    disable_existing_loggers: False
+    formatters:
+        console:
+            format: '[%(asctime)s] %(levelname)s - %(message)s'
+        dev: 
+            format: '[%(asctime)s] {{%(filename)s:%(lineno)d}} %(levelname)s - %(message)s'
+    handlers:
+        console:
+            level: INFO
+            formatter: console
+            class: logging.StreamHandler
+            stream: ext://sys.stdout
+        local:
+            level: WARN
+            formatter: dev
+            class: logging.handlers.RotatingFileHandler
+            filename: '{BENTOML_HOME}/logs/active.log'
```

Comment:
Does the current implementation handle the `{BENTOML_HOME}` in the YAML config file?   It is supported in `bentoml.cfg` file because of this https://github.com/bentoml/BentoML/blob/7bb9a7b55efc2f96d97cf0f7955ba9580599f0db/bentoml/configuration/__init__.py#L49

For this PR, how about let's just change this example to something like `/var/log/bentoml/active.log`?

Reply:
Makes sense.